src/main/java/mate/dao/impl/ManufacturerDaoImpl.java

```diff
@@ -51,12 +52,11 @@ public Manufacturer create(Manufacturer manufacturer) {
 
     @Override
     public List<Manufacturer> getAll() {
-        String query = "SELECT * FROM manufacturers WHERE is_deleted = FALSE";
+        String query = "SELECT * FROM manufacturers WHERE deleted = FALSE";
```

Comment:
is_deleted better naming for boolean column, so return it, the same with other similar cases